packages/date-time/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -0,0 +1,277 @@
+      expect(calendarProps.highlightCurrentMonth).toBe(false);
+    });
+
+    it('renders Calendar with showWeekNumbers as false by defaut', () => {
+      expect(calendarProps.showWeekNumbers).toBe(false);
+    });
+
+    it('renders Calendar with firstWeekOfYear as FirstWeekOfYear.FirstDay by defaut', () => {
+      expect(calendarProps.firstWeekOfYear).toBe(FirstWeekOfYear.FirstDay);
+    });
+
+    it('renders Calendar with showGoToToday as true by defaut', () => {
+      expect(calendarProps.showGoToToday).toBe(true);
+    });
+  });
+
+  describe('when Calendar properties are specified', () => {
+    const value = new Date(2017, 10, 1);
+    const today = new Date(2017, 9, 31);
+    const dateTimeFormatter = {
+      formatMonthDayYear: (date: Date, strings?: ICalendarStrings) => 'm/d/y',
+      formatMonthYear: (date: Date, strings?: ICalendarStrings) => 'm/y',
+      formatDay: (date: Date) => 'd',
+      formatYear: (date: Date) => 'y'
+    };
+
+    const datePicker = shallow(
+      <DatePickerBase
+        isMonthPickerVisible={false}
+        showMonthPickerAsOverlay={true}
+        value={value}
+        today={today}
+        firstDayOfWeek={2}
+        highlightCurrentMonth={true}
+        showWeekNumbers={true}
+        firstWeekOfYear={FirstWeekOfYear.FirstFullWeek}
+        showGoToToday={false}
+        dateTimeFormatter={dateTimeFormatter}
+      />
+    );
+    datePicker.setState({ isDatePickerShown: true });
+
+    const calendarProps = datePicker.find(Calendar).props();
+
+    it('renders Calendar with same isMonthPickerVisible', () => {
+      expect(calendarProps.isMonthPickerVisible).toBe(false);
+    });
+
+    it('renders Calendar with same showMonthPickerAsOverlay', () => {
+      expect(calendarProps.showMonthPickerAsOverlay).toBe(true);
+    });
+
+    it('renders Calendar with same value', () => {
+      expect(calendarProps.value).toBe(value);
+    });
+
+    it('renders Calendar with same today', () => {
+      expect(calendarProps.today).toBe(today);
+    });
+
+    it('renders Calendar with same firstDayOfWeek', () => {
+      expect(calendarProps.firstDayOfWeek).toBe(2);
+    });
+
+    it('renders Calendar with same highlightCurrentMonth', () => {
+      expect(calendarProps.highlightCurrentMonth).toBe(true);
+    });
+
+    it('renders Calendar with same showWeekNumbers', () => {
+      expect(calendarProps.showWeekNumbers).toBe(true);
+    });
+
+    it('renders Calendar with same firstWeekOfYear', () => {
+      expect(calendarProps.firstWeekOfYear).toBe(FirstWeekOfYear.FirstFullWeek);
+    });
+
+    it('renders Calendar with same showGoToToday', () => {
+      expect(calendarProps.showGoToToday).toBe(false);
+    });
+
+    it('renders Calendar with same dateTimeFormatter', () => {
+      expect(calendarProps.dateTimeFormatter).toBe(dateTimeFormatter);
+    });
+  });
+
+  describe('when date boundaries are specified', () => {
+    const defaultDate = new Date('Dec 15 2017');
```

Comment:
I would suggest using numbers in the ctor, in case someone runs the tests in a non-English system and they happen to fail.

Reply:
I'll take note of these and mark them as items todo a little later- this is a direct copy-paste of the existing DatePicker control in office-ui-fabric-react. :)